src/components/profile/Preview/style.module.scss

```diff
@@ -2,7 +2,9 @@
   align-items: center;
   display: flex;
   flex-direction: column;
+  position: sticky;
```

Comment:
One issue with this is that if the bio is long and the screen is short, the user has to scroll to the bottom of the page to scroll to the bottom of the preview. Is this fine?